lib/vscode/src/vs/server/node/connection.ts

```diff
@@ -27,7 +27,7 @@ export abstract class Connection {
 	) {
 		this.logger = logger.named(
 			this.name,
-			field("token", this.protocol.options.reconnectionToken),
+			field('token', this.protocol.options.reconnectionToken),
```

Comment:
what's the difference between `"` and `'` in JavaScript/TypeScript?

Reply:
None but VS Code's linting rules enforce `'` for "unlocalized" strings so the only time they are allowed is in calls like:

```
localize('projectQuickPick.help', "TypeScript help")
```

Reply:
ah very interesting pattern, so they're using something that extracts all `"`-quoted strings into resource bundles?

Reply:
I think they might be doing something like that, although if they can already detect and report to the dev when a string should be using `"` or `'` I'm not sure why they can't just use that knowledge to extract it and don't bother the dev. :stuck_out_tongue: